doc/releases/nautilus.rst

```diff
@@ -320,29 +315,34 @@ Instructions
    and verify that each monitor has both a ``v2:`` and ``v1:`` address
    listed.
 
-#. For each host that has been upgrade, you should update your
-   ``ceph.conf`` file so that it references both the v2 and v1
-   addresses.  Things will still work if only the v1 IP and port are
-   listed, but each CLI instantiation or daemon will need to reconnect
-   after learning the monitors real IPs, slowing things down a bit and
+#. For each host that has been upgraded, you should update your
+   ``ceph.conf`` file so that it either specifies no monitor port (if
+   you are running the monitors on the default ports) or references
+   both the v2 and v1 addresses and ports explicitly.  Things will
+   still work if only the v1 IP and port are listed, but each CLI
+   instantiation or daemon will need to reconnect after learning the
+   monitors also speak the v2 protocol, slowing things down a bit and
    preventing a full transition to the v2 protocol.
 
    This is also a good time to fully transition any config options in
-   ceph.conf into the cluster's configuration database.  On each host,
-   you can use the following command to import any option into the
+   ``ceph.conf`` into the cluster's configuration database.  On each host,
+   you can use the following command to import any options into the
```

Comment:
might want to link this note to `doc/rados/configuration/msgr2.rst` 's `Updating ceph.conf and mon_host` section as well. it features some examples,  so user can follow them.